src/lib/organisms/Table/Row/col.js

```diff
@@ -3,10 +3,10 @@ import classnames from 'classnames';
 import isEqual from 'react-fast-compare';
 
 import { guid, stopEvent, copyToClipboard } from 'utils';
-import Icon from '../../../atoms/Icon';
-import SkeletonLoader from '../../../atoms/SkeletonLoader';
-import Tooltip from '../../../molecules/Tooltip';
-import callAfterDelay from '../../../../utils/callAfterDelay';
+import Icon from 'src/lib/atoms/Icon';
+import SkeletonLoader from 'src/lib/atoms/SkeletonLoader';
+import Tooltip from 'src/lib/molecules/Tooltip';
+import callAfterDelay from 'src/utils/callAfterDelay';
```

Comment:
This path is the absolute path of the `callAfterDelay ` module to change it to alias import change it to 
`import { callAfterDelay } from 'utils';`